src/misc/ErrorsPanel/ErrorsPanel.js

```diff
@@ -17,50 +17,65 @@ import useStyles from './style';
 
 export const ErrorsPanel = (props) => {
   const classes = useStyles();
-  const { errors, labels, onClear, buildErrorsCountLabel } = props;
+  const { errors, maxErrorsCount, labels, onClear, buildErrorsCountLabel } = props;
 
   let errorsCountLabel = labels.errorsCountLabel;
   if (buildErrorsCountLabel) {
-    errorsCountLabel = buildErrorsCountLabel(errors.length);
+    errorsCountLabel = buildErrorsCountLabel(errors.length, maxErrorsCount);
   }
 
+  let errorsBounded;
+  if (errors.length > maxErrorsCount) {
+    errorsBounded = errors.slice(0, maxErrorsCount);
+  }
+
+  const getErrors = () => {
+    const errorsToRender = errorsBounded ?? errors;
+
+    return <>{errorsToRender.map((error, index) => getErrorAccordion(error, index))}</>;
```

Comment:
Could we simplify this line to `return <>{errorsToRender.map(getErrorAccordion)}</>;` ?